README.md

```diff
@@ -1,24 +1,54 @@
 # Java Extension Pack
 
-These are some of the extensions to make Java development easier and fun. 
+Java Extension Pack is a collection of popular extensions that can help write, test and debug Java applications in Visual Studio Code. Check out [Java in VS Code](https://code.visualstudio.com/docs/languages/java) to get started.
 
-# Extensions Included
+## Extensions Included
 
-- [Language Support for Java(TM) by Red Hat ](https://marketplace.visualstudio.com/items?itemName=redhat.java) - Provides Java ™ language support via Eclipse ™ JDT Language Server, which utilizes Eclipse ™ JDT, M2Eclipse and Buildship.
+By installing Java Extension Pack, the following extensions are installed:
 
-![Language Support for Java(TM) by Red Hat](https://raw.githubusercontent.com/Microsoft/vscode-java-pack/master/vscode-java.gif)
+- [📦 Language Support for Java™ by Red Hat ](https://marketplace.visualstudio.com/items?itemName=redhat.java)
+    - Code Navigation
+    - Auto Completion
+    - Refactoring
+    - Code Snippets
+- [📦 Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug)
+    - Debugging
```

Comment:
- Run & Debug Java program

Reply:
Here we need a noun to be consistent with others.